lib/dev-server.js

```diff
@@ -24,6 +24,7 @@ async function devServer(options) {
     app.use(nollupDevMiddleware(app, config, {
         hot: options.hot,
         verbose: options.verbose,
+        headers: options.headers,
```

Comment:
Looking at Webpack, they apply custom headers to all assets with their headers option. Custom headers shouldn't only apply to the content Nollup generates, but also to all of the static files that are being served. Not sure what the cleanest way of doing this with Express is, but seems to be needed to make this feature complete and fully working.

Reply:
Right, didnt consider that. I'll have a look at it later, unless you are eager to finish it 👍🏻

Reply:
Your choice, I don't mind merging and finishing it. :)

Reply:
I don't mind you finishing it :)